src/main/kotlin/no/nav/tilleggsstonader/sak/behandling/vent/SettPåVentDto.kt

```diff
@@ -0,0 +1,34 @@
+package no.nav.tilleggsstonader.sak.behandling.vent
+
+import java.time.LocalDate
+
+data class SettPåVentDto(
+    val årsaker: List<ÅrsakSettPåVent>,
+    val frist: LocalDate,
+    val kommentar: String?,
+)
+
+data class OppdaterSettPåVentDto(
+    val årsaker: List<ÅrsakSettPåVent>,
+    val frist: LocalDate,
+    val kommentar: String?,
+    val oppgaveVersjon: Int,
```

Comment:
Burde vi ha med versjon på `SettPåVentDto` også? Generelt burde vi egentlig bruke det når vi oppdaterer oppgaver 🤔

Reply:
Litt enig egentlige.Samtidig er jeg usikker på om vi har bruk for den i det hele tatt nå 👀

Hvis behandlingen ikke er på vent nå, og man setter den på vent så oppdaterer man bare oppgaven med frist og tilordnet saksbehandler. Behandlingen er på en måte den som styrer om den kan settes på vent eller ikke. Henter då oppgaven og oppdaterer den.

❗ Vid oppdatering så er det fristen/kommentar som blir oppdatert. Her kan man vel egentlige si at det burde være versjonen på settPåVent og versjonen på oppgaven som burde kontrolleres 👀 

Ved ta av vent trenger man det ikke, då sjekker man bare at settPåVent finnes, og tar den og oppgaven av vent

Reply:
Jeg skjønner. La det bare være